From 3d3c4a6acb264d7b46d975b5eba29c5521afcfbd Mon Sep 17 00:00:00 2001
From: Dan Dennedy <dan@dennedy.org>
Date: Tue, 1 Dec 2020 22:10:51 -0800
Subject: [PATCH 2/4] fix autoconf fails without opencv

Also fixes the OpenCV-based plugins actually building and installing when
OpenCV present.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 4ddc06e..7c95b9a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -98,9 +98,8 @@ PKG_CHECK_MODULES([OPENCV], [opencv4 >= 4.0.0],
    [AC_DEFINE([HAVE_OPENCV], true, [compiled including OpenCV >= 4])],
    [PKG_CHECK_MODULES([OPENCV], [opencv > 1.0.0],
        [AC_DEFINE([HAVE_OPENCV], true, [compiled including OpenCV < 4])
-    ])
+    ], [true])
 ])
-AM_CONDITIONAL([HAVE_OPENCV], [test x$HAVE_OPENCV = xtrue])
 if test ! "x$OPENCV_LIBS" = "x"; then
     HAVE_OPENCV=true
     AC_DEFINE(OPENCV_DATA_DIR,[${prefix}/share/opencv],opencv data prefix)
@@ -108,6 +107,7 @@ if test ! "x$OPENCV_LIBS" = "x"; then
     AC_SUBST([OPENCV_CFLAGS])
     AC_SUBST([OPENCV_LIBS])
 fi
+AM_CONDITIONAL([HAVE_OPENCV], [test x$HAVE_OPENCV = xtrue])
 AC_SUBST(HAVE_OPENCV)
 
 HAVE_GAVL=false
-- 
2.23.0

